### ISSUE TITLE
gegl_0_4 fails to build on darwin blocking the gimp test in nixpkgs-20.09-darwin

### ISSUE THREAD

**Issue by worldofpeace** [NixOS/nixpkgs#99108]
**Describe the bug**
```
WARNING: 'nm' does not work. Relinking will always happen on source changes.
error: nm: invalid argument --
Usage: nm [-agnopruUmxjlfAP[s segname sectname] [-] [-t format] [[-arch <arch_flag>] ...] [file ...]

[165/727] Generating Gegl-0.4.gir with a custom command
clang-7: warning: -lm: 'linker' input unused [-Wunused-command-line-argument]
clang-7: warning: -lm: 'linker' input unused [-Wunused-command-line-argument]
g-ir-scanner: link: clang -o /private/tmp/nix-build-gegl-0.4.26.drv-0/gegl-0.4.26/build/tmp-introspectcx1fbu2g/Gegl-0.4 /private/tmp/nix-build-gegl-0.4.26.drv-0/gegl-0.4.26/build/tmp-introspectcx1fbu2g/Gegl-0.4.o -L. -Wl,-rpath,. -L/private/tmp/nix-build-gegl-0.4.26.drv-0/gegl-0.4.26/build/gegl -Wl,-rpath,/private/tmp/nix-build-gegl-0.4.26.drv-0/gegl-0.4.26/build/gegl -L/nix/store/vrfcl5r2hqsp22gq85srmw0drdwaxkb8-babl-0.1.80/lib -Wl,-rpath,/nix/store/vrfcl5r2hqsp22gq85srmw0drdwaxkb8-babl-0.1.80/lib -L/nix/store/vnr4dw52bb5r529jiqda6wdqq0cabzj0-glib-2.64.5/lib -Wl,-rpath,/nix/store/vnr4dw52bb5r529jiqda6wdqq0cabzj0-glib-2.64.5/lib -L/nix/store/79sfhc2gg7y0l6k4xaaclmzj0q7r5s3d-gobject-introspection-1.64.1/lib -Wl,-rpath,/nix/store/79sfhc2gg7y0l6k4xaaclmzj0q7r5s3d-gobject-introspection-1.64.1/lib -lgegl-0.4 -lbabl-0.1 -lglib-2.0 -lgio-2.0 -lgobject-2.0 -lm -lgmodule-2.0 -lgirepository-1.0 -L/nix/store/vnr4dw52bb5r529jiqda6wdqq0cabzj0-glib-2.64.5/lib -lgio-2.0 -lgobject-2.0 -lgmodule-2.0 -lglib-2.0
[167/727] Generating gegl-0.4.vapi with a custom command
FAILED: gegl/gegl-0.4.vapi 
/nix/store/zz3np2x4mp6cas29aqv00xcb5lkkhqia-vala-0.48.7/bin/vapigen --quiet --library=gegl-0.4 --directory=/private/tmp/nix-build-gegl-0.4.26.drv-0/gegl-0.4.26/build/gegl --metadatadir=. --metadatadir=/private/tmp/nix-build-gegl-0.4.26.drv-0/gegl-0.4.26/gegl /private/tmp/nix-build-gegl-0.4.26.drv-0/gegl-0.4.26/build/gegl/Gegl-0.4.gir
Gegl-0.4.gir:7855.7-7855.60: warning: Signal `Gegl.Node.progress' conflicts with method of the same name
Gegl-0.4.gir:1770.59-1770.59: error: The symbol `Babl' could not be found
Gegl-0.4.gir:2810.57-2810.57: error: The symbol `Babl' could not be found
Gegl-0.4.gir:12176.55-12176.55: error: The symbol `Babl' could not be found
Gegl-0.4.gir:12180.57-12180.57: error: The symbol `Babl' could not be found
ninja: build stopped: subcommand failed.
```

gegl build: https://hydra.nixos.org/build/127967622
gimp build: https://hydra.nixos.org/build/127967748
nixpkgs-20.09-darwin: https://hydra.nixos.org/job/nixpkgs/nixpkgs-20.09-darwin/darwin-tested#tabs-constituents

cc @jtojnar 

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: gegl_0_4 gimp
# a list of nixos modules affected by the problem
module:
```

**Comment by jtojnar** [comment]
This should be fixed in gegl master https://gitlab.gnome.org/GNOME/gegl/-/merge_requests/83

**Comment by vcunat** [comment]
`gimp` on Hydra trunk shows _Last successful build 2020-06-10 08:16:17_, so I think we could "safely" drop it from channel-blocking packages for 20.09 (and even newer).

**Comment by worldofpeace** [comment]
> `gimp` on Hydra trunk shows _Last successful build 2020-06-10 08:16:17_, so I think we could "safely" drop it from channel-blocking packages for nixpkgs-20.09-darwin (and even newer).
> 
> EDIT: still, if applying the linked patch fixes it by itself...

TBH I think the channels need to have constituents for some applications like this. `gimp` on darwin sounds like a reasonable deliverable.

**Comment by worldofpeace** [comment]
fixed in #99112
